src/Serializers/FingerprintSerializer.php

```diff
@@ -97,4 +108,19 @@ private function serializeValuesPerLanguageArray( $array ) {
 		return $serialization;
 	}
 
+	private function serializeAliasGroup( AliasGroup $aliasGroup, array &$serialization ) {
+		$language = $aliasGroup->getLanguageCode();
+		foreach( $aliasGroup->getAliases() as $value ) {
+			$result = array(
+				'language' => $language,
+				'value' => $value
+			);
+			if ($aliasGroup instanceof AliasGroupFallback) {
+				$result['language'] = $aliasGroup->getActualLanguageCode();
+				$result['source'] = $aliasGroup->getSourceLanguageCode();
+			}
+			$serialization[$language][] = $result;
```

Comment:
I find this method (and the same for terms above) hard to read. I will try to refactor this in a later patch.